specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-04-01-preview/examples/workspaceManagerConfigurations/CreateOrUpdateWorkspaceManagerConfiguration.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "api-version": "2023-04-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "workspaceManagerConfigurationName": "default",
+    "workspaceManagerConfiguration": {
+      "properties": {
+        "mode": "Enabled"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/workspaceManagerConfigurations/default",
+        "etag": "\"3f6451dd-1b58-4bef-bce7-72eba6b354d7\"",
+        "name": "default",
+        "type": "Microsoft.SecurityInsights/workspaceManagerConfigurations",
+        "properties": {
+          "mode": "Enabled"
```

Comment:
>mode [](http://example.com/codeflow?start=11&length=4)

one feedback I suppose you might not be able to take is that 'mode' 'Enabled' may be quite unclear in meaning

Reply:
Thanks for the feedback!